docs/v2/4_erc20-interactions.md

```diff
@@ -88,7 +94,7 @@ public class ERC20NativeBalanceOfExample : MonoBehaviour
 
 ### Name {#name}
 
-Returns the name of the token, e.g. "USD Coin"
+Returns the name of an ERC-20 token associated with a specified contract address (e.g. "USD Coin").
```

Comment:
Replace "USD Coin" with "USDC"

Reply:
<img width="179" alt="image" src="https://user-images.githubusercontent.com/36713098/221094663-98fa20ed-d942-477e-b4b0-af280c39c414.png">
If I read as proxy the name of the contract address at https://goerli.etherscan.io/address/0x07865c6e87b9f70255377e024ace6630c1eaa37f#readProxyContract, it reads "USD Coin" rather than "USDC", which I guess is the symbol. Thoughts? @waymobetta

Reply:
So in all cases then when using this contract it is to be indicated as USD Coin instead of USDC?

Reply:
Cleared with @waymobetta. name and symbol are separate params
![image](https://user-images.githubusercontent.com/36713098/221240792-5f94b2ad-b219-4ecf-905e-2db84d60c79b.png)